bundles/org.openhab.voice.mimictts/README.md

```diff
@@ -0,0 +1,45 @@
+# Mimic Text-to-Speech
+
+Mimic (version 3 and above) is an offline open source Text-To-speech engine designed by Mycroft A.I. for the eponym Vocal Assistant.
+It provides multiple voices, available in different languages and variants.
+Its neural network is built upon some very good and some not-so-good models. Try some to be sure you get the best one for your need.
+Mimic3 doesn't need Mycroft, it can be run standalone as a service or command line utility.
+When launched as a web server, it exposes its capability through a web API. This TTS bundle make use of these feature, so please take note : this openHAB TTS bundle is NOT a standalone ! it requires mimic web server to run somewhere (on your openHAB computer, or your network)
```

Comment:
Maybe you can add a link to the documentation explaining how to install this WEB server and the different voices.

Reply:
Yes, good idea, I did some rewrite and include it.